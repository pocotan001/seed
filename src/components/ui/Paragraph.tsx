import Text from "./Text";

const Paragraph = Text.withComponent("p");

export default Paragraph;
