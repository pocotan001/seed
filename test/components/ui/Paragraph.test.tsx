import * as React from "react";
import * as renderer from "react-test-renderer";
import Paragraph from "~/components/ui/Paragraph";

describe("<Paragraph>", () => {
  it("should render correctly", () => {
    const tree = renderer.create(<Paragraph>alo</Paragraph>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
