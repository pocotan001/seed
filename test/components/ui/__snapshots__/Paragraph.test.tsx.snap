// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph> should render correctly 1`] = `
<p
  className="Text-sc-18djoaj-0-p gFwNcq"
>
  alo
</p>
`;
